geonetwork/variant-postgres.md

```diff
@@ -8,20 +8,25 @@ In order to setup the connection from geonetwork, you **must** inject the follow
 
 If your postgres instance is listening on a non-standard port, you must also set that variable: - `POSTGRES_DB_PORT`: postgres port on your database server (defaults to `5432`)
 
-### Linking to a postgres container
+### Connecting to a postgres container
 
-Linking to a postgres container, is pretty straightforward: - `--link <some-postgres-container>:postgres`
+Conecting to a postgres container, is pretty straightforward: you just need to make sure that they are running in the same network. For instance, you can create a bridge network:
 
-For instance, if you want to run the official image for postgres, you could launch it like this:
+```console
+$ docker network create --driver bridge mynet
+```
+
+Then if you want to run the official image for postgres, you could launch it like this:
 
 ```console
-$ docker run --name some-postgres -p 5432:5432 -d postgres
+$ docker run --name postgres --network=mynet -d postgres
 ```
+In order to enable geonetwork to discover it, **make sure you name it `postgres`**.
```

Comment:
Wouldn't it be more flexible for users to instead recommend that they use `--network-alias postgres` so that it's more clear they can name their container whatever is useful for them?

Reply:
Also, isn't there an environment variable they can use on the `geonetwork` container to adjust which hostname it attempts to connect to?

Reply:
@tianon you are right: I simplified the explanation a bit, by using the $POSTGRES_DB_HOST variable in both use cases (IP addresses and container names) 👍 . Please have a look at my latest [PR] (https://github.com/docker-library/docs/pull/1087). Thanks!